src/js/services/WebsiteService.ts

```diff
@@ -54,6 +55,13 @@ class WebsiteService {
         texts: batch.map(item => {
           // This is temporary solution
           let isSEO = false
+          let elementSeoItem = TranslationElementCandidates.get(item.tagName)
```

Comment:
ko šis kods risina?

Reply:
Te ir apšaubāms vai pareizi jo tag name nav atribūta nosaukums

Reply:
tā kā tur jau tā bija pirms tam pārbaude vai item ir seo tips, tad pievienoju vienu papildus check, kas paskatās, vai item tags ir iekļauts TranslationElementCandidates sarakstā, kur elementi ir SEO; ja ir iekļauts, tātad tas arīdzan ir jāsaglabā kā SEO (isSEO = true)

Reply:
> Te ir apšaubāms vai pareizi jo tag name nav atribūta nosaukums

īstenībā, to attributeName tad nemaz nevajag, izņēmu. pietiek ar description. attributeName ir svarīgs attribute seo elementiem.

Reply:
Vajadzētu risināt šo loģikas jautājumu iepriekš DOMTranslation vai kaut kur citur

Reply:
iznesu loģiku ārā no WebsiteService